Meet error during make progress

**Describe the bug**

`Distributions/qif.cpp:72:81: error: no matching function for call to ‘QStringList::QStringList(<brace-enclosed initializer list>)’`

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/45301505/183390961-7d456acd-2baa-45dc-b0b5-f17db5251954.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 18.04
 - arch: aarch64
 - arm-linux-gnueabihf-gcc 7
 - qt 5.9.0
 - qt-installer-framework 3.2.2 build from source 

**Additional context**
I built qif from source 3.2.2 and used "make install" to install it. But when I use "qmake -r", it told me that "QtInstallerFramework not found! use binaries from PATH"
I have tried to add "CONFIG += c++11", but nothing helps.

--- a/Deploy/Distributions/qif.cpp
+++ b/Deploy/Distributions/qif.cpp
@@ -69,7 +69,7 @@ QList<SystemCommandData> QIF::runCmd() {
     } else {
         auto commandsList = binarycreator.split(' ');
         cmd.command = commandsList.first();
-        cmd.arguments = QStringList{commandsList.begin() + 1, commandsList.end()};
+        cmd.arguments = commandsList.mid(1,-1);
     }
 
     auto location = DeployCore::_config->getTargetDir() + "/" + getLocation();

--- a/Deploy/extracter.cpp
+++ b/Deploy/extracter.cpp
@@ -10,7 +10,6 @@
 #include "pluginsparser.h"
 #include "configparser.h"
 #include "metafilemanager.h"
-#include "pathutils.h"
 #include <QCoreApplication>
 #include <QDebug>
 #include <QDir>

--- a/QIFData/InstallerBase.pri
+++ b/QIFData/InstallerBase.pri
@@ -30,7 +30,7 @@ message(DEPLOYER = $$DEPLOYER)
 BINARY_LIST
 REPO_LIST
 
-sopprted_versions = 4.1 4.0 3.2 3.1 3.0
+sopprted_versions = 4.5 4.4 4.3 4.2 4.1 4.0 3.2 3.1 3.0
 for(val, sopprted_versions) {
 
     exists( $$QT_DIR/../../../Tools/QtInstallerFramework/$$val/bin/ ) {
@@ -41,7 +41,7 @@ for(val, sopprted_versions) {
 }
 
 isEmpty (BINARY_LIST) {
-      warning( "QtInstallerFramework not found! use binaries from PATH" )
+      warning( "QtInstallerFramework not found! use binaries from PATH." )
       EXEC=binarycreator
       REPOGEN=repogen
 

